### ISSUE TITLE
Firefox stable does not render meshes w/ objectAlpha setting

### ISSUE THREAD

**Issue by falkben** [google/neuroglancer#140]
On Firefox stable (`67.0.1 (64-bit)`), meshes with `objectAlpha` parameter don't render (though segmentations do).  Firefox developer edition and Chrome have no problems. (Linux).

Firefox stable developer console shows the following warnings (on viz.neurodata.io where we have the sourcemaps):

```
Error: WebGL warning: drawArrays: Float32 blending requires EXT_float_blend. offscreen.ts:317:7
Error: WebGL warning: drawArrays: Float32 blending requires EXT_float_blend.
Error: WebGL warning: drawElements: Float32 blending requires EXT_float_blend. frontend.ts:211:7
```

Link to reproduce: https://neuroglancer-demo.appspot.com/#!%7B%22layers%22:%5B%7B%22source%22:%22precomputed://https://s3.amazonaws.com/zbrain/atlas_owen%22%2C%22type%22:%22segmentation%22%2C%22objectAlpha%22:0.94%2C%22segments%22:%5B%221%22%2C%2210%22%2C%2211%22%2C%2212%22%2C%2213%22%2C%2214%22%2C%2215%22%2C%2216%22%2C%2217%22%2C%2218%22%2C%2219%22%2C%222%22%2C%2220%22%2C%2221%22%2C%2222%22%2C%2223%22%2C%2224%22%2C%2225%22%2C%2226%22%2C%2227%22%2C%2228%22%2C%2229%22%2C%223%22%2C%2230%22%2C%2231%22%2C%2232%22%2C%2233%22%2C%224%22%2C%225%22%2C%226%22%2C%227%22%2C%228%22%2C%229%22%5D%2C%22skeletonRendering%22:%7B%22mode2d%22:%22lines_and_points%22%2C%22mode3d%22:%22lines%22%7D%2C%22name%22:%22zbrain_atlas%22%7D%5D%2C%22navigation%22:%7B%22pose%22:%7B%22position%22:%7B%22voxelSize%22:%5B798%2C798%2C2000%5D%2C%22voxelCoordinates%22:%5B585.5321044921875%2C321.09796142578125%2C71.52733612060547%5D%7D%7D%2C%22zoomFactor%22:1490.8602740309134%7D%2C%22showDefaultAnnotations%22:false%2C%22perspectiveOrientation%22:%5B0.7711568474769592%2C-0.2878552973270416%2C0.13006828725337982%2C-0.5527555346488953%5D%2C%22perspectiveZoom%22:11789%2C%22showSlices%22:false%2C%22selectedLayer%22:%7B%22layer%22:%22zbrain_atlas%22%7D%2C%22layout%22:%224panel%22%7D

Possibly related to: #138

**Comment by jbms** [comment]
I'm not able to reproduce on Firefox 67.0 on Linux.  Additionally, according to this page, enabling EXT_color_buffer_float is supposed to also enable EXT_float_blend: https://developer.mozilla.org/en-US/docs/Web/API/EXT_float_blend

**Comment by jbms** [comment]
Also note that this extension does not seem to exist on Chrome.

We could attempt to enable this extension in webgl/context.ts and simply not give an error if the extension is unavailable, but it would be good to find out what is happening here.

**Comment by falkben** [comment]
I'm not sure what is going on - tested on Chrome and Firefox on MacOS and it works okay as well.

I just turned off all my extensions to see if somehow I had a conflict there and there was no change.  Also tried in private browsing mode with no effect either. Adjusting the opacity 3D slider to go to 100% renders the meshes.  it's just when that slider is less than 100%.

It's not a critical thing from our perspective, just something I noticed today.  If Firefox Developer edition works, perhaps the issue will resolve itself with time as beta -> standard?
